haas/client/user.py

```diff
@@ -50,3 +50,22 @@ def remove(self, user, project):
         return self.check_response(
                 self.httpClient.request("POST", url, data=payload)
                 )
+
+    def set_admin(self, username, privilege):
```

Comment:
@knikolla I kept the privilege string to mirror the `create()` command (which also accepts a string).  The documentation for `create()` mentions that a user is either 'admin' or 'regular', which maps to true or false. 

I'm not sure if we'd rather go with consistency or with what might make more sense.